app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterClaimController.scala

```diff
@@ -67,9 +66,18 @@ class EnterClaimController @Inject() (
 
   implicit val dataExtractor: DraftC285Claim => Option[ClaimsAnswer] = _.claimsAnswer
 
-  def startClaim(): Action[AnyContent] =
+  def aaa(taxCategory: TaxCategory, taxCode: TaxCode): Action[AnyContent] = Action {
```

Comment:
what is this method for?

Reply:
I made a mistake at merging, so I rolled back my merge commit and cleanup.
Removed it again.